src/ReportManager/DetailsSection/index.js

```diff
@@ -234,6 +236,12 @@ const DetailsSection = ({
       <button ref={submitFormButtonRef} type="submit" />
     </Form>}
 
+    {!!formSchema && isEFBSchema &&
+      <SchemaFormContextProvider schema={formSchema} onFieldChange={onFormChange} formData={reportForm.event_details} onFormSubmit={onFormSubmit} >
```

Comment:
Why don't we have the implementation of the context provider inside the `SchemaForm` component? We can send all these props to the `SchemaForm` and inject them to the context value there:
```
// SchemaForm/index.js
const SchemaForm = ({ ... }) => {
  ...
  return <SchemaFormContextProvider value={{ ... }}>
    ...
  </SchemaFormContextProvider>;
};
```

This way the parent doesn't have to implement the context, which is cleaner.